vagrant/aws/aws-init.sh

```diff
@@ -18,16 +18,31 @@
 # or bring up your mini Kafka cluster.
 
 # Install dependencies
-sudo apt-get install -y maven openjdk-6-jdk build-essential \
-            ruby-dev zlib1g-dev realpath python-setuptools
+sudo apt-get install -y \
+  vagrant \
+  maven \
+  openjdk-8-jdk \
```

Comment:
Well. It did not work for jdk-6 for me, so I updated. Also, this guide recommends to use "Ubuntu Server" and jdk-8 is available there. If anyone uses a different OS, she must not expect that all scripts works as intended IMHO.